pkg/rpc/client/rpc_test.go

```diff
@@ -508,6 +508,27 @@ var rpcClientTestCases = map[string][]rpcClientTestCase{
 			},
 		},
 	},
+	"getstateroot": {
```

Comment:
A test for error case can also be added to `rpcClientErrorCase` in order to cover error after `c.PerformRequest`.